build/squire-raw.js

```diff
@@ -889,6 +889,28 @@ var deleteContentsOfRange = function ( range, root ) {
     return frag;
 };
 
+var isNodeEmpty = function ( node ) {
+    if (!node) {
+        return false;
+    }
+
+    // check for image nodes
+    if ( node.nodeType === Node.ELEMENT_NODE && node.nodeName === 'IMG' ) {
```

Comment:
I couldn't find anything in the Selectors API similar to jquery's addBack() either. However I would just combine the ifs like  `if ( node.nodeName === 'IMG' || node.querySelectory( 'img' ) )`
and then just drop the nodeType check.